rootfs/usr/bin/hwctl

```diff
@@ -16,12 +16,12 @@ function system-info {
 	fi
 
 	# CPU
-	cat /proc/cpuinfo > /tmp/cpuinfo.txt
-	CPU_VENDOR=$(grep 'vendor_id' /tmp/cpuinfo.txt | head -n1 | cut -d':' -f2 | sed 's/^ //')
-	CPU_NAME=$(grep 'model name' /tmp/cpuinfo.txt | head -n1 | cut -d':' -f2 | sed 's/^ //')
-	CPU_PHYSICAL=$(grep 'cpu cores' /tmp/cpuinfo.txt | head -n1 | cut -d':' -f2 | tr -d ' ')
-	CPU_SPEED="$(lscpu | grep 'CPU max MHz' | head -n 1 | cut -d':' -f2 | tr -d ' ' | cut -d'.' -f1)"
-	CPU_ARCH="$(lscpu | grep 'Architecture:' | head -n 1 | cut -d':' -f2 | tr -d ' ')"
+	lscpu > /tmp/lscpu.txt
+	CPU_ARCH="$(grep 'Architecture:' /tmp/lscpu.txt | head -n 1 | cut -d':' -f2 | tr -d ' ')"
+	CPU_NAME=$(grep 'Model name' /tmp/lscpu.txt | head -n1 | cut -d':' -f2 | sed 's/^[[:space:]]*//')
+	CPU_VENDOR=$(grep 'Vendor ID' /tmp/lscpu.txt | head -n1 | cut -d':' -f2 | sed 's/^[[:space:]]*//')
+	CPU_PHYSICAL=$(ls /sys/bus/cpu/devices | wc -l)
```

Comment:
On my machine that I tested with, this gives me 32, which is the number of threads. My machine only has 16 physical cores, which is what was previously outputted.